libavpipe/src/avpipe_xc.c

```diff
@@ -3186,8 +3186,19 @@ get_filter_str(
         char *filt_buf = NULL;
         int filt_buf_size;
         int filt_str_len;
-        const char* filt_template =
-            "[in] scale=%d:%d [in-1]; movie='%s', setpts=PTS [over]; [in-1] setpts=PTS [in-1a]; [in-1a][over]  overlay='%s:%s:alpha=0.1' [out]";
+
+        /*
+         * Create an overlay filter that expects the image be sized for the source video - apply the
+         * overlay first and then scale the result.
+         *
+         * - movie='%s': overlay image, assumed same size as input
+         * - scale=%d:%d: output resolution
+         *
+         * Previous filter:
+         * "[in] scale=%d:%d [in-1]; movie='%s', setpts=PTS [over]; [in-1] setpts=PTS [in-1a]; [in-1a][over]  overlay='%s:%s:alpha=0.1' [out]";
+         */
+        const char * filt_template =
+            "movie='%s', setpts=PTS[ov]; [in][ov] overlay=%s:%s:format=auto:alpha=0.1, scale=%d:%d [out]";
```

Comment:
This makes a lot more sense than the previous ordering

We don't have an alpha parameter?

Reply:
Currently the image gets preprocessed and has its alpha setting.  I am not sure why  alpha=0.1 here but I kept it for this round so I minimize changes.